src/InputNumber.tsx

```diff
@@ -457,12 +457,12 @@ const InputNumber = React.forwardRef(
     };
 
     // ========================== Controlled ==========================
-    // Input by precision
+    // Input by precision & formatter
     useUpdateEffect(() => {
       if (!decimalValue.isInvalidate()) {
         setInputValue(decimalValue, false);
```

Comment:
This will make setInputValue being triggered every time cause `formatter` is a inline function.